api/src/main/resources/liquibase-update-to-2.1.xml

```diff
@@ -35,5 +35,256 @@
+			<not>
+				<sqlCheck expectedResult="0">
+					SELECT COUNT(*) from cohort_member
+				</sqlCheck>
+			</not>
+		</preConditions>
+		<comment>Updating cohort_member.date_created with value cohort.date_created</comment>
+		<update tableName="cohort_member">
+			<column name="date_created" type="DATETIME"
+					valueComputed="(select date_created from cohort where cohort_member.cohort_id = cohort.cohort_id)"/>
+		</update>
+	</changeSet>
+	<changeSet author="vshankar" id="201610131530-1">
+		<preConditions>
+			<not>
+				<columnExists tableName="cohort_member" columnName="voided"/>
+			</not>
+		</preConditions>
+		<comment>Adding column cohort_member.voided</comment>
+		<addColumn tableName="cohort_member">
+			<column name="voided" type="BOOLEAN">
+				<constraints nullable="false"/>
+			</column>
+		</addColumn>
+	</changeSet>
+	<changeSet author="vshankar" id="201610131530-1.1">
+		<preConditions onFail="MARK_RAN">
+			<columnExists tableName="cohort_member" columnName="voided"/>
+		</preConditions>
+		<comment>Adding defaultValue for cohort_member.voided</comment>
+		<addDefaultValue tableName="cohort_member" columnName="voided" defaultValueBoolean="false"/>
+	</changeSet>
+	<changeSet author="vshankar" id="201610131530-2">
+		<preConditions>
+			<not>
+				<columnExists tableName="cohort_member" columnName="voided_by"/>
+			</not>
+		</preConditions>
+		<comment>Adding column cohort_member.voided_by</comment>
+		<addColumn tableName="cohort_member">
+			<column name="voided_by" type="int"/>
+		</addColumn>
+	</changeSet>
+	<changeSet author="vshankar" id="201610131530-3">
+		<preConditions>
+			<not>
+				<columnExists tableName="cohort_member" columnName="date_voided"/>
+			</not>
+		</preConditions>
+		<comment>Adding column cohort_member.date_voided</comment>
+		<addColumn tableName="cohort_member">
+			<column name="date_voided" type="DATETIME"/>
+		</addColumn>
+	</changeSet>
+	<changeSet author="vshankar" id="201610131530-4">
+		<preConditions>
+			<not>
+				<columnExists tableName="cohort_member" columnName="void_reason"/>
+			</not>
+		</preConditions>
+		<comment>Adding column cohort_member.void_reason</comment>
+		<addColumn tableName="cohort_member">
+			<column name="void_reason" type="varchar(255)"/>
+		</addColumn>
+	</changeSet>
+	<changeSet id="201610042145-2" author="vshankar">
+		<preConditions>
+			<not>
+				<columnExists tableName="cohort_member" columnName="uuid"/>
+			</not>
+		</preConditions>
+		<comment>Adding column cohort_member.uuid</comment>
+		<addColumn tableName="cohort_member">
+			<column name="uuid" type="char(38)">
+				<constraints nullable="false"/>
+			</column>
+		</addColumn>
+	</changeSet>
+	<changeSet id="201610042145-2.1" author="vshankar">
+		<preConditions onFail="MARK_RAN"><or><dbms type="mysql"/><dbms type="oracle" /></or></preConditions>
+		<comment>Generating UUIDs for all rows in cohort_member table via built in uuid function.</comment>
+		<update tableName="cohort_member" >
```

Comment:
I did run and confirmed that it does not work. :)
I created some cohort and cohort members before running your changes. The cohort member id field was properly filled but uuid remained null.